tests/Unit/Evolution/Systems/ScalarWave/Test_EnergyDensity.cpp

```diff
@@ -21,9 +21,29 @@
 
 namespace {
 template <size_t SpatialDim, typename DataType>
-void test_compute_item_in_databox() noexcept {
+void test_compute_item_in_databox(const DataType& used_for_size) noexcept {
   TestHelpers::db::test_simple_tag<ScalarWave::Tags::EnergyDensity<SpatialDim>>(
       "EnergyDensity");
+  TestHelpers::db::test_compute_tag<
+      ScalarWave::Tags::EnergyDensityCompute<SpatialDim>>("EnergyDensity");
+
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> dist(-1., 1.);
+
+  const Scalar<DataVector> pi = make_with_random_values<Scalar<DataVector>>(
+      make_not_null(&generator), dist, used_for_size);
+  const tnsr::i<DataVector, SpatialDim, Frame::Inertial> phi =
+      make_with_random_values<tnsr::i<DataVector, SpatialDim, Frame::Inertial>>(
+          make_not_null(&generator), dist, used_for_size);
+
+  const auto box = db::create<
+      db::AddSimpleTags<ScalarWave::Pi, ScalarWave::Phi<SpatialDim>>,
+      db::AddComputeTags<ScalarWave::Tags::EnergyDensityCompute<SpatialDim>>>(
+      pi, phi);
+
+  const auto expected = ScalarWave::energy_density(pi, phi);
+  CHECK_ITERABLE_APPROX(
+      (db::get<ScalarWave::Tags::EnergyDensity<SpatialDim>>(box)), expected);
 }
```

Comment:
please add a blank line between the functions. It makes the code easier to read :)